README.md

```diff
@@ -109,6 +113,63 @@ constant:
 MyModel = Avromatic::Model.model(schema_name :my_model)
 ```
 
+#### Experimental: Union Support
+
+Avromatic contains experimental support for unions containing more than one
+non-null member type. This feature is experimental because Virtus attributes
+may attempt to coerce between types too aggressively.
```

Comment:
Are there more specifics we can add here so users can tell if the feature will work for them?